README.md

```diff
@@ -75,52 +94,92 @@ Please refer to our [CONTRIBUTING](CONTRIBUTING.md) document.
-  - `config` holds configuration parameters.  These include parameters used locally by the node as well as parameters which must be agreed upon by the protocol.
+  - `crypto` contains the cryptographic constructions we're using for hashing,
+    signatures, and VRFs. There are also some Algorand-specific details here
+    about spending keys, protocols keys, one-time-use signing keys, and how they
+    relate to each other.
+  - `config` holds configuration parameters.  These include parameters used
+    locally by the node as well as parameters which must be agreed upon by the
+    protocol.
   - `data` defines various types used throughout the codebase.
-	 - `basics` holds basic types such as MicroAlgos, account data, and addresses.
-     - `account` defines accounts, including "root" accounts (which can spend money) and "participation" accounts (which can participate in the agreement protocol).
-	 - `transactions` defines transactions that accounts can issue against the Algorand state.  These include standard payments and also participation key registration transactions.
-	 - `bookkeeping` defines blocks, which are batches of transactions atomically committed to Algorand.
-	 - `pools` implements the transaction pool.  The transaction pool holds transactions seen by a node in memory before they are proposed in a block.
-	 - `committee` implements the credentials that authenticate a participating account's membership in the agreement protocol.
-  - `ledger` ([README](ledger/README.md)) contains the Algorand Ledger state machine, which holds the sequence of blocks.  The Ledger executes the state transitions that result from applying these blocks.  It answers queries on blocks (e.g., what transactions were in the last committed block?) and on accounts (e.g., what is my balance?).
-  - `protocol` declares constants used to identify protocol versions, tags for routing network messages, and prefixes for domain separation of cryptographic inputs.  It also implements the canonical encoder.
-  - `network` contains the code for participating in a mesh network based on websockets. Maintains connection to some number of peers, (optionally) accepts connections from peers, sends point to point and broadcast messages, and receives messages routing them to various handler code (e.g. agreement/gossip/network.go registers three handlers).
-     - `rpcs` contains the HTTP RPCs used by `algod` processes to query one another.
-  - `agreement` ([README](agreement/README.md)) contains the agreement service, which implements Algorand's Byzantine Agreement protocol.  This protocol allows participating accounts to quickly confirm blocks in a fork-safe manner, provided that sufficient account stake is correctly executing the protocol.
-  - `node` integrates the components above and handles initialization and shutdown.  It provides queries into these components.
+	 - `basics` holds basic types such as MicroAlgos, account data, and
+           addresses.
```

Comment:
Nit: the second line here is not aligned with the first bulleted line. (Same with the following bullets.)

Reply:
In the current version, basics, account, transactions, etc are listed as sub-bullets of data. 
Hence, these are not aligned here. 

I would like to confirm which sub-bullets are correct, and which ones need to be fixed. 

I see there is a problem in my change for daemon/algod/api, but the hierarchy of the other bullets are not changed.

Reply:
Ah, I was pointing out something much more mundane--I think the file mixes spaces and tabs, which is affecting how the text is aligned.

The sub-bullets look right to me.